api/tools/src/main/java/org/pmiops/workbench/tools/SetAuthority.java

```diff
@@ -31,6 +32,9 @@ private Set<Authority> commaDelimitedStringToAuthoritySet(String str) {
       if (cleanedValue.isEmpty()) {
         continue;
       }
+      if (ALL_AUTHORITIES.equals(cleanedValue)) {
+        return new HashSet<>(Arrays.asList(Authority.values()));
```

Comment:
We never want to grant ALL, so it is worth adding enforcement?  Maybe in run() rather than here.

Reply:
I'm not going to say it's impossible, but it should be rare. I'm imagining a rare testing scenario where you might want to add ALL, then remove DEVELOPER to test certain interactions.

At the very least though, I added a user confirmation speedbump in this situation in the Ruby tools